video_creation/screenshot_downloader.py

```diff
@@ -51,7 +56,40 @@ def download_screenshots_of_reddit_posts(reddit_object, screenshot_num):
                 '[data-click-id="text"] button'
             ).click()  # Remove "Click to see nsfw" Button in Screenshot
 
-        page.locator('[data-test-id="post-content"]').screenshot(path="assets/temp/png/title.png")
+        page.locator('[data-test-id="post-content"]').screenshot(
+            path="assets/temp/png/title.png"
+        )
+
+        # translate code
+
+        if getenv("POSTLANG"):
+            print_substep("Translating post...")
+            texts_in_tl = ts.google(reddit_object["thread_title"], to_language=os.getenv("POSTLANG"))
+
+            img = Image.open("assets/temp/png/title.png")
+        
+            width = img.size[0] 
+            height = img.size[1]
+
+            d1 = ImageDraw.Draw(img)
+            font = ImageFont.truetype("arial.ttf", 22)
+        
+            wrapper = textwrap.TextWrapper(width=50)
+            wrapped_str = wrapper.fill(text=texts_in_tl)
+
+            if (getenv("THEME").upper() == "DARK"):
+                fillmode = "#1a1a1b"
+                textmode = (255, 255, 255)
+            else:
+                fillmode = "whiite"
```

Comment:
Misspelling `fillmode = "whiite"` change it to white

```suggestion
                fillmode = "white"
```

```suggestion
                fillmode = "#ffffff"
```